lib/feature/browser/browser_tabs_view/browser_start_view.dart

```diff
@@ -64,14 +80,49 @@ class _BrowserStartViewState extends State<BrowserStartView> {
             searchText: searchText,
           );
 
-    return ColoredBox(
-      color: colors.appBackground,
-      child: CustomScrollView(
-        slivers: slivers,
+    return RepaintBoundary(
+      key: _globalKey,
+      child: ColoredBox(
+        color: colors.appBackground,
+        child: CustomScrollView(
+          slivers: slivers,
+        ),
       ),
     );
   }
 
+  Future<void> _saveScreenshot() async {
+    final now = DateTime.now();
+    if (_lastScreenshotTime != null &&
+        now.difference(_lastScreenshotTime!) < _screenShotPeriodDuration) {
+      return;
+    }
+
+    _lastScreenshotTime = now;
+
+    WidgetsBinding.instance.addPostFrameCallback((_) async {
+      try {
+        final boundary = _globalKey.currentContext?.findRenderObject()
+            as RenderRepaintBoundary?;
+
+        final image = await boundary!.toImage(pixelRatio: 2);
+
+        final byteData = await image.toByteData(format: ui.ImageByteFormat.png);
+        final pngBytes = byteData?.buffer.asUint8List().toList();
+        if (pngBytes == null) {
+          throw Exception("Can't convert image to bytes");
+        }
+
+        final file = File(BrowserTab.defaultImagePath);
+        await file.writeAsBytes(pngBytes);
+
+        await FileImage(File(BrowserTab.defaultImagePath)).evict();
+      } catch (e, s) {
+        _log.severe('Error while saving screenshot: $e', null, s);
```

Comment:
Why `$e` in 1-st param if second param of `severe` must contain error itself

Reply:
```dart
  void severe(Object? message, [Object? error, StackTrace? stackTrace]) =>
      log(Level.SEVERE, message, error, stackTrace);
```

So, I just put message and stack trace to the logs here.